### DISCUSSION TITLE
【深度许愿】 关于文件拖放到桌面的优化建议

### DISCUSSION THREAD

**Discussion by lizipeng0013** [linuxdeepin/developer-center#4232] [Features Request & Ideas | 特性请求 & 头脑风暴]
![202208070027392421_录屏_dde-desktop_20220807002523](https://user-images.githubusercontent.com/123794418/234304256-dfe29b64-8caa-4ec7-bef0-125e168bf83d.gif)
现在需要先拖到桌面，然后才能移动位置

如果能直接拖到想放置的位置就更好了

**Comment by BLumia** [comment]
cc @meiyixiang

**Comment by lizipeng0013** [comment]
> ![202208070027392421_录屏_dde-desktop_20220807002523](https://user-images.githubusercontent.com/123794418/234304256-dfe29b64-8caa-4ec7-bef0-125e168bf83d.gif) [ ![202208070027392421_录屏_dde-desktop_20220807002523](https://user-images.githubusercontent.com/123794418/234304256-dfe29b64-8caa-4ec7-bef0-125e168bf83d.gif) ](https://user-images.githubusercontent.com/123794418/234304256-dfe29b64-8caa-4ec7-bef0-125e168bf83d.gif) [ ](https://user-images.githubusercontent.com/123794418/234304256-dfe29b64-8caa-4ec7-bef0-125e168bf83d.gif) 现在需要先拖到桌面，然后才能移动位置
> 
> 如果能直接拖到想放置的位置就更好了

还有，想把文件拖到桌面的快捷方式打开也有问题，只支持桌面的文件，其它文件夹的一拖就把文件拖到桌面了
